### PR TITLE
Add 'Move to Waiting List button'

### PR DESCRIPTION
**Pull request by FinnIckler**
The backend was already there so I just needed to add the button

### CODE REVIEW

File: Frontend/src/pages/registration_administration/components/RegistrationActions.jsx [hunk 1]
```diff
@@ -87,20 +103,45 @@ export default function RegistrationActions({ selected, refresh }) {
...
+                [
+                  ...selected.pending,
+                  ...selected.cancelled,
+                  ...selected.accepted,
+                ],
+                'waiting_list'
+              )
+            }
+          >
+            <UiIcon name="hourglass" /> Delete
```
Comment (kr-matthews) [inline Frontend/src/pages/registration_administration/components/RegistrationActions.jsx:132]: This should be "Send to Waitlist" or something, not "Delete".

File: Frontend/src/pages/registration_administration/components/RegistrationActions.jsx [hunk 2]
```diff
@@ -87,20 +103,45 @@ export default function RegistrationActions({ selected, refresh }) {
...
               changeStatus(
-                [...selected.pending, ...selected.cancelled],
+                [
+                  ...selected.pending,
+                  ...selected.accepted,
+                  ...selected.waiting,
+                ],
                 'cancelled'
               )
             }
```
Comment (kr-matthews) [inline Frontend/src/pages/registration_administration/components/RegistrationActions.jsx:147]: It won't let me comment 2 lines below, but the "Delete" should probably be "Cancel"?

File: Frontend/src/pages/registration_administration/components/RegistrationActions.jsx [hunk 3]
```diff
@@ -87,20 +103,45 @@ export default function RegistrationActions({ selected, refresh }) {
...
+                [
+                  ...selected.accepted,
+                  ...selected.cancelled,
+                  ...selected.waiting,
+                ],
                 'pending'
               )
             }
           >
             <UiIcon name="times" /> Reject
```
Comment (kr-matthews) [inline Frontend/src/pages/registration_administration/components/RegistrationActions.jsx:115]: "Reject" also seems unclear to me. I could image it sending them to any of the non-approved lists...

**Comment by FinnIckler** [comment]
ah yeah, I changed all those names but forgot to push the commit...

**Pull request merged**